apps/demo/package.json

```diff
@@ -11,11 +11,13 @@
     "check": "astro check && tsc --noEmit && astro build",
     "build": "astro build",
     "preview": "astro preview",
-    "astro": "astro"
+    "astro": "astro",
+    "clean": "rm -rf .turbo dist"
   },
   "dependencies": {
-    "@astro-reactive/form": "file:packages/form",
-    "@astro-reactive/validator": "file:packages/validator",
+    "@astro-reactive/form": "*",
+    "@astro-reactive/validator": "*",
+    "@astro-reactive/common": "*",
```

Comment:
Let's not manually add the `common` package as dependency here; It should be automatically added when `form` or `validator` is installed and we want to catch the scenario if this fails :)

Reply:
I Agree. `common` should be an internal dependencies of `validator` and `form` package.

Reply:
I guess `common` can be set `private: true` in it's  `package.json` as it only used internally and doesn't need to be published.

Reply:
I think private packages are not  available for free tier in organizations on npmjs.org